lib/initCookieSession.js

```diff
@@ -1,30 +1,13 @@
 const cookieSession = require('cookie-session');
 const config = require('./config');
-//Cookie max age variant for environments 30mins prod 2 elseware
-//TODO make config var?
 //Set to expire 1min before apiAuth token lasts under licence_data scope.
-const maxAge = 84924000; //23hrs 59mins //(config.NODE_ENV === 'production')? 30 : 2;
-//
+const maxAge = 84924000; //23hrs 59mins
 
 const sessionDetails = {
 	name: 'FTKat',
-		keys: [config.get('KMT_SECRET')],
-		domain: config.get('HOST'),
-		maxAge: maxAge
+	keys: [config.KMT_SECRET],
+	domain: config.HOST,
+	maxAge: maxAge
 };
 
 module.exports = cookieSession(sessionDetails);
```

Comment:
We have this set up in the `.editorconfig` (https://github.com/Financial-Times/kmt-utilities/blob/master/.editorconfig#L9). But I don't see the flagging :)